bin/mochii.js

```diff
@@ -23,45 +23,16 @@ async function run(args) {
   runMochitests(args);
 }
 
-/*
- * updateArgs is a small convenience method for determining which tests to run
- * if there are no args, include all the tests
- * if there is no browser_dbg prefix, add it.
- */
-function updateArgs(args) {
-  if (args.length == 0) {
-    return ["devtools/client/debugger/new"];
-  }
-
-  const _args = args.slice(0, -1);
-  const maybeFile = args[args.length - 1];
-  const hasFile = maybeFile && !maybeFile.includes("/");
+const argString = process.argv[0].includes("bin/node")
+  ? process.argv.slice(2)
+  : process.argv;
 
-  if (!hasFile) {
-    return args;
-  }
-
-  const file = maybeFile;
-  if (file.includes("browser_dbg")) {
-    return args;
-  }
```

Comment:
this drops the hacky version of turning `expressions` into `browser_dbg-expressions`. It's hacky because 

1. it assumes the last part of the args is the test file.
2. it assumes you don't have many test files...

i use this a bunch because i can never remember `browser_dbg-`  like is the hyphen first? but i think it needs some cleaning up and it shouldn't be blocking